device_attributes/index.bs

```diff
@@ -0,0 +1,200 @@
+<pre class='metadata'>
+Title: Device Attributes API
+Shortname: device-attributes
+Level: 1
+Editor: Anqing Zhao 132057, Google LLC https://www.google.com, anqing@google.com
+Status: w3c/CG-DRAFT
+URL: https://wicg.github.io/WebApiDevice/device_attributes
+Abstract: This document defines a web platform API that enables developers to query device information (device ID, serial number, location, etc) from managed devices. This API is invaluable for such use cases as Virtual Desktop Infrastructure (VDI) and Context-based Configuration.
+Repository: https://github.com/WICG/WebApiDevice
+Group: wicg
+</pre>
+ 
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+spec:url; type:dfn; text:origin
+spec:html; type:dfn; for:/; text:global object
+spec:webidl; type:dfn; text:resolve
+spec:webidl; type:dfn; text:record
+</pre>
+ 
+<pre class="anchors">
+ 
+</pre>
+ 
```

Comment:
You should have an introduction section to help readers get into the right context. Feel free to do that or flesh this out in another PR.

```suggestion
 # Introduction # {#introduction}
 
 The Device Attributes API allows [=device administrators=] to allow certain origins to access certain attributes of the device running the UA. Unlike [[MANAGED-CONFIG]], which allows the [=device administrator=] to pass a custom JSON object to each origin, this API expects to pass the same set of values to all origins that the administrator has enabled.
```